examples/va/nvidia-mdev/README.md

```diff
@@ -70,6 +70,102 @@ metadata:
   namespace: openstack
 ```
 
+#### Openstack Dataplane Composable Service
+
+An Openstack Dataplane service can used to customize how the GPU cards need to
+be installed on the EDPM nodes. An example of the service can be seen below.
+With the OSDPS configured, the operator would need to make sure to include
+the service to the list of services for the Openstack Dataplane NodeSet.
+**Note:** This is not an officially supported procedure for installing Nvidia
+GPU's in RHOSO and is meant to be purely an example of how to leverage OSDPS.
+Please reference Nvidia's documentation when creating a procedure to install
+GPU's.
+
+```YAML
```

Comment:
This content (the edpm service with the embedded playbook) already lives in `va/nvidia-mdev/edpm/nodeset/nova_sriov.yaml` so instead of pasting it here could you just link to it?

Reply:
Yes that is a better flow, will update.